### ISSUE TITLE
general: Disconnected: Connection Closed   al login

### ISSUE THREAD

**Issue by shorewall** [tgalal/yowsup#2644]
I had yowsup working fine for a long time up to about 10 days ago.
Now same fault on three different Linux boxes (MAgeia 5 Mageia 6 Mageia 6.1)
Yowsup 2.5.7

**To Reproduce**
1. yowsup-cli demos --debug -l "39331xxxxxxxxxxx:yxxxxyyyyyyyyyyyM84=" -y -E s40 (or -E android)
2.  /L to login

See attached debug log for details

**Comment by shorewall** [comment]
[Yow.log](https://github.com/tgalal/yowsup/files/2861776/Yow.log)

**Comment by shorewall** [comment]
I forgot to write that re-registering the number and getting a new password is ok. so my number doesn't seems to be banned. But nothing changes.

**Comment by bigounours** [comment]
Same problem,
I registred with Android, then replace "android" by "s40" and recompile... 
My number is not ban, I can connect with a phone.

**Comment by r0bocharov** [comment]
I have the same issue. I have registered successfully with android env and then tried login with android/s40 and both failed. My debug output looks similar:

```
DEBUG:yowsup.layers.logger.layer:tx:
<stream:features>
</stream:features>

DEBUG:yowsup.layers.logger.layer:tx:
<auth passive="true" user="31616xxxxxx" mechanism="WAUTH-2">
</auth>

DEBUG:yowsup.layers.network.layer:Disconnected, reason: Connection Closed
```

**Comment by r0bocharov** [comment]
I've tried register and then login with the latest updated evn_android.py, but no luck.

```
_MD5_CLASSES = "bdZVQh58MKZaaeiYnh7wkw=="
_VERSION = "2.19.51"
```

**Comment by alexvong1995** [comment]
@shorewall
Yowsup has stopped working since 23 Jan 2019. See #2635 for details.
I think this issue can be closed since it is the same as #2635.

**Comment by shorewall** [comment]
Ok, I'll follow https://github.com/tgalal/yowsup/issues/2635 for a solution.
Tks
